lib/UserAgentApplication.ts

```diff
@@ -841,6 +850,7 @@ namespace Msal {
         acquireTokenPopup(scopes: Array<string>, authority: string): Promise<string>;
         acquireTokenPopup(scopes: Array<string>, authority: string, user: User): Promise<string>;
         acquireTokenPopup(scopes: Array<string>, authority: string, user: User, extraQueryParameters: string): Promise<string>;
+        @resolveTokenOnlyIfOutOfIframe
```

Comment:
@austindyoung Remove this decorator. This decorator cannot be applied to loginPopup and acquireTokenPopup as in case of a popup window, window.parent is same as window. You should have this decorator only for the acquireTokenSilentMethod method. All other changes look fine.